packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -233,13 +238,11 @@ class TextStyle extends Diagnosticable {
   /// The color to use when painting the text.
   final Color color;
 
-  /// The name of the font to use when painting the text (e.g., Roboto).
+  /// The name of the font to use when painting the text (e.g., Roboto). If the
+  /// font is defined in a package, this will be prefixed with
+  /// 'packages/package_name/' (e.g. 'packages/cool_fonts/Roboto').
```

Comment:
You should add some text saying that this prefixing is done by the constructor when you provide the packages argument, so that people don't think they have to do it.